Lib/pydoc_data/topics.py

```diff
@@ -919,7 +919,7 @@ class and instance attributes applies as for regular assignments.
 levels, one for multiplicative operators and one for additive
 operators:
 
-   m_expr: u_expr | m_expr "*" u_expr | m_expr "@" m_expr |
+   m_expr: u_expr | m_expr "*" u_expr | m_expr "@" u_expr |
```

Comment:
```suggestion
   m_expr: u_expr | m_expr "*" u_expr | m_expr "@" m_expr |
```
Please revert this, this is generated during the release process.

Reply:
Thank I have reverted the change.